cmd/meroxa/root/apps/deploy.go

```diff
@@ -217,7 +225,7 @@ func (d *Deploy) uploadSource(ctx context.Context, appPath, url string) error {
 
 	var buf bytes.Buffer
 
-	if d.lang == JavaScript || d.lang == Python {
+	if d.lang == Python {
```

Comment:
turbine-js does not need to track temp dir because there is no temp dir building anymore! 😄